alma/alma.php

```diff
@@ -503,9 +563,52 @@ public function viewAccess()
         Tools::redirectAdmin($location);
     }
 
+    /**
+     * @return mixed|null
+     */
     public function getContent()
     {
-        return $this->runHookController('getContent', null);
+        $hasPSAccount = $this->renderPSAccount();
+
+        return $this->runHookController('getContent', ['hasPSAccount' => $hasPSAccount]);
+    }
+
+    /**
+     * @return bool
+     */
+    public function renderPSAccount()
+    {
+        if (!$this->checkCompatibilityPSModule()) {
+            return false;
+        }
+
+        try {
+            $accountsFacade = $this->getService('alma.ps_accounts_facade');
+            $accountsService = $accountsFacade->getPsAccountsService();
+        } catch (\PrestaShop\PsAccountsInstaller\Installer\Exception\InstallerException $e) {
+            $accountsInstaller = $this->getService('alma.ps_accounts_installer');
+            $accountsInstaller->install();
+            $accountsFacade = $this->getService('alma.ps_accounts_facade');
+            $accountsService = $accountsFacade->getPsAccountsService();
+        }
+
+        try {
+            $mediaHelper = new \Alma\PrestaShop\Helpers\MediaHelper();
```

Comment:
Maybe we can add this injection in the constructor ?

Reply:
Done